playwright/frame.py

```diff
@@ -289,21 +328,17 @@ async def selectOption(
         values: ValuesToSelect,
         timeout: int = None,
         noWaitAfter: bool = None,
-    ) -> None:
-        return await self._channel.send(
-            "selectOption",
-            dict(
-                selector=selector,
-                values=convertSelectOptionValues(values),
-                timeout=timeout,
-                noWaitAfter=noWaitAfter,
-            ),
-        )
+    ) -> List[str]:
+        params = locals_to_params(locals())
+        if "values" in params:
+            values = params.pop("values")
+            params = dict(**params, **convert_select_option_values(values))
```

Comment:
nit: `{**params, **convert_select_option_values(values)}`
But probably just a personal preference.

Reply:
I did not know that {} was more Python-ic! Happy to move across the board!